src/components/ToggleSwitch.js

```diff
@@ -1,85 +1,62 @@
 import React from 'react'
 import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
 
-export default function ToggleSwitch({ toggled, onChange }) {
-  return <Switch id="test-switch" toggled={toggled} onChange={onChange} />
+ToggleSwitch.propTypes = {
+  selected: PropTypes.bool.isRequired,
+  toggleSelected: PropTypes.func.isRequired,
 }
 
-const Switch = ({ id, toggled, onChange }) => {
+export default function ToggleSwitch({ selected, toggleSelected }) {
   return (
-    <>
-      <SwitchContainer>
-        <SpendingRed>Spending</SpendingRed>
-        <HiddenSwitchInput
-          className="switch-checkbox"
-          id={id}
-          type="checkbox"
-          checked={toggled}
-          onChange={onChange}
-        />
-        <SwitchLabel toggled={toggled} className="switch-label" htmlFor={id}>
-          <SwitchButton className="switch-button"></SwitchButton>
-        </SwitchLabel>
-        <IncomeGreen>Income</IncomeGreen>
-      </SwitchContainer>
-    </>
+    <ToggleWrapper>
+      <ToggleContainer onClick={toggleSelected}>
+        <DialogueButton className={selected ? '' : 'disabled'}>
+          {selected ? 'Income' : 'Spending'}
+        </DialogueButton>
+      </ToggleContainer>
+    </ToggleWrapper>
   )
 }
 
-const SwitchContainer = styled.div`
+const ToggleWrapper = styled.div`
   display: flex;
   justify-content: center;
 `
-const SpendingRed = styled.div`
-  color: red;
-  font-weight: bold;
-`
-const IncomeGreen = styled.div`
-  color: green;
-  font-weight: bold;
-`
-const toggleColors = {
-  true: '#36ffa1', //refers to income
-  false: '#ff7a78', //refers to spending
-}
-
-const HiddenSwitchInput = styled.input`
-  height: 0;
-  width: 0;
-  visibility: hidden;
+const ToggleContainer = styled.div`
+  position: relative;
+  width: 124px;
+  height: 36px;
+  margin: 10px;
+  border-radius: 3px;
+  padding: 2px;
+  background-color: #c4c4c4;
```

Comment:
If you like you can save your colors as CSS-variables and give them names. Then it's a little bit easier to guess what it will look like.